src/app/domains/views/DomainDetails/DomainDetailsHeader/AddRecordForm/AddRecordForm.test.tsx

```diff
@@ -1,45 +1,41 @@
-import { mount } from "enzyme";
-import { act } from "react-dom/test-utils";
+import { screen, render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { Provider } from "react-redux";
 import { MemoryRouter } from "react-router-dom";
 import { CompatRouter } from "react-router-dom-v5-compat";
 import configureStore from "redux-mock-store";
 
-import AddRecordForm from "./AddRecordForm";
+import { Labels as RecordFieldsLabels } from "../../../../components/RecordFields/RecordFields";
```

Comment:
This should probably be an absolute import.

Reply:
I'll get that changed quickly. Think it's a couple places where that happens